types/gamedig/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "module": "commonjs",
+        "module": "node16",
```

Comment:
any reason for using node16 specifically?
I have no inherent issue with this, just curious.

Reply:
the original issue was that the types didnt work with `moduleResolution: node16`: https://github.com/gamedig/node-gamedig/issues/569  

and the config was changed here to have a test for it:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69767#issuecomment-2179507649